### PR TITLE
changed static images to show animations in onboarding

### PR DESCRIPTION
**Pull request by pbochenski**
### Task

 - [JIRA](https://netguru.atlassian.net/browse/CRAI-280)
 
### Description
changed static images to show animations in onboarding
 
### Checklist

 - [x] I am following the [code style guide](https://netguru.atlassian.net/wiki/display/ANDROID/Android+best+practices)
 - [x] The code includes tests of new features
 - [x] The code passes static analysis
 - [x] README.md is up to date
 - [x] Version number is up to date
 - [x] ProGuard configuration files are up to date
 - [x] All temporary TODOs and FIXMEs are removed
 - [x] I have tested the solution on these devices:
  * Samsung galaxy s9
  
### Merge

 - [x] Committer
 - [ ] Reviewer
 - [ ] Project lead

### CODE REVIEW

File: app/src/main/kotlin/co/netguru/android/carrecognition/feature/onboarding/PageFragment.kt [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    abstract fun getResourceUri(): Uri
+
+    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View? {
+        return inflater.inflate(R.layout.onboarding_inside_fragment, container, false)
+    }
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+
+        if (!fragmentResume && fragmentVisible){   //only when first time fragment is created
```
Comment (anyashka) [inline app/src/main/kotlin/co/netguru/android/carrecognition/feature/onboarding/PageFragment.kt]: Space is missing :)

File: app/src/main/kotlin/co/netguru/android/carrecognition/feature/onboarding/OnboardingDiscoverFragment.kt [hunk 2]
```diff
@@ -1,22 +1,18 @@
...
 import kotlinx.android.synthetic.main.onboarding_inside_fragment.*
 
-class OnboardingDiscoverFragment : Fragment() {
+class OnboardingDiscoverFragment : PageFragment() {
 
-    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View? {
-        return inflater.inflate(R.layout.onboarding_inside_fragment, container, false)
-    }
+    override fun getResourceUri(): Uri =
+        Uri.parse("android.resource://" + context?.packageName + "/" + R.raw.onboarding_2)
```
Comment (Arazjel) [inline app/src/main/kotlin/co/netguru/android/carrecognition/feature/onboarding/OnboardingDiscoverFragment.kt]: `"android.resource://${context?.packageName}/${R.raw.onboarding_2}"`

File: app/src/main/kotlin/co/netguru/android/carrecognition/feature/onboarding/OnboardingRecognizeFragment.kt [hunk 3]
```diff
@@ -1,31 +1,27 @@
...
 
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         super.onViewCreated(view, savedInstanceState)
-        carImg.setImageResource(R.drawable.img_onboarding_1)
         titleTxt.setText(R.string.recognize_cars)
         descriptionTxt.setText(R.string.point_the_camera)
     }
 
+    override fun getResourceUri(): Uri =
+        Uri.parse("android.resource://" + context?.packageName + "/" + R.raw.onboarding_1)
```
Comment (Arazjel) [inline app/src/main/kotlin/co/netguru/android/carrecognition/feature/onboarding/OnboardingRecognizeFragment.kt]: as above

File: app/src/main/kotlin/co/netguru/android/carrecognition/feature/onboarding/OnboardingStayUpdatedFragment.kt [hunk 4]
```diff
@@ -1,31 +1,27 @@
...
 
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         super.onViewCreated(view, savedInstanceState)
-        carImg.setImageResource(R.drawable.img_onboarding_3)
         titleTxt.setText(R.string.stay_updated)
         descriptionTxt.setText(R.string.expect_updates)
     }
 
+    override fun getResourceUri(): Uri =
+        Uri.parse("android.resource://" + context?.packageName + "/" + R.raw.onboarding_3)
```
Comment (Arazjel) [inline app/src/main/kotlin/co/netguru/android/carrecognition/feature/onboarding/OnboardingStayUpdatedFragment.kt]: as above

File: app/src/main/res/layout/onboarding_inside_fragment.xml [hunk 5]
```diff
@@ -22,8 +21,8 @@
         android:layout_above="@+id/descriptionTxt"
         android:layout_marginBottom="@dimen/margin_large"
         android:gravity="center"
-        android:paddingEnd="@dimen/margin_xxxlarge"
         android:paddingStart="@dimen/margin_xxxlarge"
+        android:paddingEnd="@dimen/margin_xxxlarge"
         android:textColor="@color/black"
         android:textSize="20sp" />
```
Comment (Arazjel) [inline app/src/main/res/layout/onboarding_inside_fragment.xml]: move text size to dimens?

**Pull request merged**